### PR TITLE
[script.module.requests] 2.15.1

### PR DESCRIPTION
**Pull request by Freso**
### Description

* Update [`requests`](https://github.com/kennethreitz/requests) Python library to version 2.15.1 for Kodi 13/Gotham–16/Jarvis. See upstream [`HISTORY.rst`](https://github.com/kennethreitz/requests/blob/v2.15.1/HISTORY.rst) for upstream changes.

This is the last upstream release with bundled dependencies/packages, making it the last version to be packageable for Kodi 13–16 without adding a number of additional Python module add‐ons to this repository.

I have taken over maintenance of this add‐on from @beenje. See https://github.com/beenje/script.module.requests/pull/24.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

### CODE REVIEW

File: script.module.requests/addon.xml [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+       provider-name="kennethreitz, beenje, Freso">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
   </requires>
   <extension point="xbmc.python.module"
              library="lib" />
   <extension point="xbmc.addon.metadata">
-    <summary>Python HTTP for Humans</summary>
-    <description>Packed for KODI from https://github.com/kennethreitz/requests</description>
+    <summary lang="en_GB">Python HTTP for Humans</summary>
```
Comment (Rechi) [inline script.module.requests/addon.xml]: `lang="en"`, gotham isn't compatible with ` lang="en_GB"`

File: script.module.requests/addon.xml [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+    <description lang="en_GB">Packed for KODI from https://github.com/kennethreitz/requests</description>
```
Comment (Rechi) [inline script.module.requests/addon.xml]: `lang="en"`, gotham isn't compatible with ` lang="en_GB"`

File: script.module.requests/addon.xml (hunk 1 above)
Comment (Freso) [reply script.module.requests/addon.xml]: Ah, thanks!

File: script.module.requests/addon.xml (hunk 2 above)
Comment (Freso) [reply script.module.requests/addon.xml]: Fixed now. Thanks again! :)

File: script.module.requests/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+*.pyc
```
Comment (Rechi) [inline script.module.requests/.gitignore]: please remove this file

**Pull request merged**